src/ui/line/item.rs

```diff
@@ -426,7 +426,7 @@ impl Item {
                         let mut chars = vec![];
                         for ch in span.content.chars() {
                             // NOTE: all control characters (except "\n") should have been removed via the `Printable` trait
-                            // and "\n" should have been removed when building Spans from the item
```

Comment:
This change is incorrect, or? The comment talks about the original `Vec<Span>`? I think this is the reason `Spans` has been renamed.

Reply:
Yep, I believe you're right - since this method converts the `Vec<Vec<Span>>` to `Vec<Line>` - you can leave this as `Spans` or `Vec<Span>`, etc.